google-cloud-pubsub/samples/topics.rb

```diff
@@ -290,6 +290,37 @@ def publish_messages_async_with_concurrency_control topic_id:
   # [END pubsub_publisher_concurrency_control]
 end
 
+def publish_messages_async_with_flow_control topic_id:
+  # [START pubsub_publisher_flow_control]
+  # topic_id = "your-topic-id"
+  require "google/cloud/pubsub"
+
+  pubsub = Google::Cloud::Pubsub.new
+
+  topic = pubsub.topic topic_id, async: {
+    # Configure how many messages the publisher client can hold in memory
+    # before publishing succeeds, and what to do when messages exceed the limit.
+    flow_control: {
+      message_limit: 100,
+      byte_limit: 10 * 1024 * 1024, # 10 MiB
+      # Block more messages from being published when the limit is reached. The
+      # other options are :ignore and :error.
+      limit_exceeded_behavior: :block
+    }
+  }
+  # Publish 1000 messages in quick succession to trigger publisher flow control.
```

Comment:
I wouldn't say "to trigger flow control." It's possible that flow control will be triggered, but it's also entirely possible it won't be if messages get out quickly enough. Either way, the fact that flow control was triggered isn't something that will be easily visible to the user. Maybe say "Rapidly publishing 1000 messages in a loop may be constrained by flow control."

Reply:
Will do, thanks!

Reply:
Updated.